web-interface/vm-consoles-modules/vm-desktop-viewer-web.adoc

```diff
@@ -0,0 +1,15 @@
+[[vm-desktop-viewer-web]]
+== Using Desktop Viewer to connect to RDP Console (Windows only)
+
+Connect with *Desktop Viewer* to running virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine.
+. Click *Consoles*. The VNC console opens by default.
+. Select *Desktop Viewer* in drop-down list.
+. Choose nework interface you want to connect to from *Network Interface* drop-down.
```

Comment:
Small changes -> "Choose the network interface you want to connect to from the *Network Interface* drop-down."